internal/pkg/skuba/cni/cilium.go

```diff
@@ -125,9 +125,10 @@ func CreateOrUpdateCiliumConfigMap(client clientset.Interface) error {
 		return err
 	}
 	ciliumConfigMapData := map[string]string{
-		"debug":        "false",
-		"disable-ipv4": "false",
-		"etcd-config":  string(etcdConfigDataByte),
+		"debug":       "false",
+		"enable-ipv4": "true",
+		"enable-ipv6": "false",
+		"etcd-config": string(etcdConfigDataByte),
```

Comment:
Please, bump the manifest version of `Cilium` in the `AddonsVersion`, so `skuba addon upgrade` will know that the manifests changed and that it has to reapply them: https://github.com/SUSE/skuba/blob/master/internal/pkg/skuba/kubernetes/versions.go#L79-L138, it can be done on all Kubernetes versions.

Reply:
ahh perfect, that was what I was looking for regarding the update part, nice!

Reply:
done!